src/librustc_mir/transform/generator.rs

```diff
@@ -91,6 +91,16 @@ impl<'tcx> MutVisitor<'tcx> for RenameLocalVisitor<'tcx> {
             *local = self.to;
         }
     }
+
+    fn visit_terminator_kind(&mut self, kind: &mut TerminatorKind<'tcx>, location: Location) {
```

Comment:
is this change necessary? We should never set `self.from` to `_0`, right?

Reply:
Ah nevermind, we do: https://github.com/jonas-schievink/rust/blob/e22cc993fbf9918784ec38a25e04564e7b1973ee/src/librustc_mir/transform/generator.rs#L233